python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/dot1x.j2

```diff
@@ -19,12 +19,13 @@
 
 #### 802.1X Global
 
-| System Auth Control | Protocol LLDP Bypass | Dynamic Authorization |
-| ------------------- | -------------------- | ----------------------|
+| System Auth Control | Protocol LLDP Bypass | Dynamic Authorization | Dropped Statistics Packets |
```

Comment:
I think this makes more sense:
```suggestion
| System Auth Control | Protocol LLDP Bypass | Dynamic Authorization | Dropped Packets Statistics |
```

Reply:
Maybe just follow the EOS CLI `Statistics Packets Dropped`